src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -16,6 +16,7 @@
         <BusinessTypeConfirm
           class="py-8 px-6"
           :class="{ 'invalid-section': getShowErrors && !businessTypeConfirmValid }"
+          :showErrors="getShowErrors && !businessTypeConfirmValid"
```

Comment:
Must use getShowErrors && !businessTypeConfirmValid otherwise error--text will remain even if checkbox is checked.

Reply:
I appreciate the PR comments but you have to call out something that is more complex than usual (something that needs explanation).